src/py/flwr/common/constant.py

```diff
@@ -119,6 +119,7 @@
 AUTH_TYPE = "auth_type"
 ACCESS_TOKEN_KEY = "access_token"
 REFRESH_TOKEN_KEY = "refresh_token"
+AUTH_TLS_CHECK = "FLWR_USER_AUTH_TLS_CHECK"
```

Comment:
In case of confuseion. Node auth also relies on TLS.

```suggestion
USER_AUTH_TLS_CHECK = "FLWR_USER_AUTH_TLS_CHECK"
```